runtime/path_convert/src/main.cpp

```diff
@@ -170,18 +175,17 @@ static const test_data datas[] = {
     ,{"foo:echo /bar/baz", "foo:echo /bar/baz", false}
     ,{"@/foo/bar", "@" MSYSROOT2 "/foo/bar", false}
     ,{"@/foo/bar@", "@" MSYSROOT2 "/foo/bar@", false}
-    ,{"'@/foo/bar'", "'@" MSYSROOT2 "/foo/bar'", false}
+    ,{"'@/foo/bar'", "'@/foo/bar'", false}
```

Comment:
I see that there might be a good chance that this new behavior causes regressions: Some MINGW programs take `@<file>` arguments (similar to [GNU `tar`'s `--file=@<file-containing-paths>`](https://www.gnu.org/software/tar/manual/tar.html#file) feature).

IIRC the intention of the backported change was to exclude `@@<...>` from being path-converted, and to continue to path-convert `@<...>` arguments.

Reply:
Note that this is only because of the leading `'` and not because of the other content, same for various other examples. I'm not sure why our tests all wrap things in `'` and under which circumstances this can occur.

Reply:
Ah, that makes sense. Good catch!